python/tvm/tir/buffer.py

```diff
@@ -187,13 +187,11 @@ def __getitem__(self, indices):
         if any(isinstance(index, slice) and index.step is None for index in indices):
             region = []
             analyzer = Analyzer()
-            for index in indices:
+            for i, index in zip(range(len(indices)), indices):
```

Comment:
What about using the following
```suggestion
            for i, index in enumerate(indices):
```

Reply:
good idea!